internal/hud/resourceview.go

```diff
@@ -217,10 +217,15 @@ func (v *ResourceView) resourceExpandedDC() rty.Component {
 	l.Add(rty.TextString(" "))
 	l.AddDynamic(rty.NewFillerString(' '))
 
-	// TODO(maia): ports
-
 	st := v.res.DockerComposeTarget().StartTime
 	if !st.IsZero() {
+		if len(v.res.Endpoints) > 0 {
+			for _, endpoint := range v.res.Endpoints {
+				l.Add(rty.TextString(endpoint))
```

Comment:
we should prooobably split this out into a helper func so that if we want to change the way we display endpoints, we only have to change it in one place? but eh.

Reply:
done! tho i don't think the abstraction helped us much